tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -123,12 +122,14 @@ class VarDictJavaEndToEnd(tumorBam: PathToBam,
                           maxThreads: Int = 32) extends Pipeline {
   import VarDictJava.BinDir
 
+  // Put these here so that they'll error at construction if missing
```

Comment:
Same thing here - moving them up to the constructor surfaces any problems sooner.